lesson1/src/stacks_queues/stack.js

```diff
@@ -1,43 +1,48 @@
 class Stack {
   constructor() {
     this.stack = [];
-    this.top = 0; // feel free to use or remove top
+    
   }
 
   /*
    Push should insert the value into the queue
    */
   push(value) {
     // todo: do stuff
+    return this.stack.push(value)
```

Comment:
give it a try to implement this ones yourself and not use the array methods (ill let you keep your queue as is .. cause that one is a bit harder), this means keeping the `top` attribute (probably should be initialized on -1 (my bad). and doing something like... 
```top ++
this.stack[top] = value
``` on the push method.. (im probably missing some checks)